components/wifi/esp8266-driver/ESP8266Interface.h

```diff
@@ -293,6 +303,18 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface {
      */
     virtual nsapi_connection_status_t get_connection_status() const;
 
+    /** Suspend the traffic of the interface
+     *
+     *  @param direction The direction going to be suspended
+     */
+    virtual void interface_suspend(uint16_t direction);
```

Comment:
suspend/resume wouldn't be sufficient (no need for interface_) ? I dont see it defined anywhere in Mbed OS codebase as it is here

Reply:
well I put the interface_ here because I don't want to confuse the users. The API is not to suspend/resume the ESP8266 itself, but only the traffic. 
But on second thought, the class itself is ESP8266Interface, so perhaps it wouldn't confuse users. How do you think?